integration/apps/rails-five/README.md

```diff
@@ -0,0 +1,94 @@
+# Rails 5: Demo application for Datadog APM
+
+A generic Rails 5 web application with some common use scenarios.
+
+For generating Datadog APM traces and profiles.
+
+## Installation
+
+Install [direnv](https://github.com/direnv/direnv) for applying local settings.
+
+1. `cp .envrc.sample .envrc` and add your Datadog API key.
+2. `direnv allow` to load the env var.
+3. `cp docker-compose.yml.sample docker-compose.yml` and configure if necessary.
+4. `docker-compose run --rm api bin/setup`
+
+## Running the application
+
+### To monitor performance of Docker containers with Datadog
+
+```sh
+docker run --rm --name dd-agent  -v /var/run/docker.sock:/var/run/docker.sock:ro -v /proc/:/host/proc/:ro -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -e API_KEY=$DD_API_KEY datadog/docker-dd-agent:latest
+```
+
+### Starting the web server
+
+Run `docker-compose up` to auto-start the webserver. It should bind to `localhost:80`.
+
+Alternatively, you can run it manually with:
+
+```sh
+docker-compose run --rm -p 80:80 api bin/dd-demo <process>
+```
```

Comment:
This doesn't work for me...? Whatever I pass in `<process>` docker does not seem to pass in to the command, so I always get puma (which is the default set elsewhere).

(Also `api` should probably be `app`)